test/integration/dredd-test.coffee

```diff
@@ -592,18 +601,27 @@ describe 'Dredd class Integration', ->
       )
     )
 
-    it('recognizes all 3 transactions', ->
-      assert.equal(matches.length, 3)
+    it('recognizes all 6 transactions', ->
+      assert.equal(matches.length, 6)
     )
     it('the transaction #1 is skipped', ->
       assert.equal(matches[0][1], 'skip')
     )
-    it('the transaction #2 is not skipped (unskipped in hooks)', ->
-      assert.notEqual(matches[1][1], 'skip')
+    it('the transaction #2 is skipped', ->
+      assert.equal(matches[1][1], 'skip')
     )
-    it('the transaction #3 is not skipped (status 200)', ->
+    it('the transaction #3 is not skipped (unskiped in hooks)', ->
       assert.notEqual(matches[2][1], 'skip')
     )
+    it('the transaction #4 is not skipped (status 200)', ->
+      assert.notEqual(matches[3][1], 'skip')
+    )
+    it('the transaction #5 is not skipped (status 200)', ->
+      assert.notEqual(matches[4][1], 'skip')
+    )
+    it('the transaction #6 is not skipped (unskiped in hooks)', ->
+      assert.notEqual(matches[5][1], 'skip')
+    )
```

Comment:
Similar to above. Make an array of expectations about the individual matches and iterate.

```coffee
[
  {statusCode: 200, skipped: false, reason: 'unskipped by default'}
  {statusCode: 500, skipped: false, reason: 'unskipped in hooks'}
  ...
].forEach((expectation, i) ->
  context("the transaction ##{i}", ->
    it("has status code #{expectation.statusCode}", ->
      assert...
    )
    it("is #{if expectation.skipped then '' else 'not '}skipped (#{expectation.reason})", ->
      assert...
    )
  )
)
```

(Just out of my head, can contain mistakes)